docs/user/01-60-enable-sidecar-injection.md

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
The file numbering will be conflicting with https://github.com/kyma-project/istio/blob/main/docs/user/01-60-recovering-from-unintentional-istio-module-removal.md